README.md

```diff
@@ -108,7 +108,7 @@ enabled = true
 # Use hashed filenames for tagged metrics instead of human readable
 # https://github.com/lomik/go-carbon/pull/225
 hash-filenames = true
-# specify to enable/disable compressed format
+# specify to enable/disable compressed format. IMPORTANT: Only one process/thread could write to compressed whisper files at a time, especially when you are rebalancing graphite clusters (with buckytools, for example), flock needs to be enabled both in go-carbon and your tooling.
```

Comment:
@azhiltsov you are right, do you think this is good enough for reminding users of this enabling flock?

Reply:
Might be additionally to that flock config parameter need to be explicitly set to true? 
what is your take on that, @lomik ?

Reply:
@lomik @azhiltsov I have make flock enabled by default in example and doc now.